settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':smi-lib-commons-model'
```

Comment:
Rename this file back to example.  It is for debugging/development across projects only.  It will break the build.